crowbar_framework/config/environment.rb

```diff
@@ -18,3 +18,4 @@
 require File.expand_path("../application", __FILE__)
 
 Rails.application.initialize!
+Rails.cache.write(:sanity_check_errors, Crowbar::Sanity.check)
```

Comment:
JFYI: I can't use the initializers to fill the cache, as the i18n is not yet available at this point